test/fixtures/rules/std_rule_cases/L019.yml

```diff
@@ -12,6 +12,24 @@ leading_comma_violations:
       b
     FROM c
 
+leading_comma_violation_with_inline_comment:
+  fail_str: |
+    SELECT
+        a
+        , b -- inline comment
+        , c
+        /* non inline comment */
+        , d
+    FROM e
+  fix_str: |
+    SELECT
+        a,
+        b ,-- inline comment
```

Comment:
The formatting looks odd here -- space before the comma, no space between the comma and the comment. Will the final output look like this, or do other lint rules/fixes typically handle this?

Reply:
Yeah, I said more about this in 3bdf0583fe78cedec1b5623ab562a43542231c36 but basically I had a hard time figuring out the formatting. The final output looks normal though, as other rules get applied. If you have some suggestions though I'm all ears 👂

Reply:
No suggestions, really, although I'm sure it's doable with more tinkering. We were discussing a similar "rule interdependencies" question on my open PR #905. I think I'm okay having other rules help out with whitespace formatting, but I'm not sure Niall and Alan are convinced. 😜

Reply:
Thanks for the heads up, let's see what they have to say 🤔

https://github.com/sqlfluff/sqlfluff/pull/905#discussion_r609607938

Reply:
Until we resolve the other conversation I'm fine with this. This isn't the only example of where there's some odd formatting around, so I don't feel the need to hold this back until that's resolved. 👍